doc-content/enterprise-only/springboot/openshift-springboot-proc.adoc

```diff
@@ -0,0 +1,167 @@
+[id='openshift-springboot-proc_{context}']
+= Running a SpringBoot business application on {OPENSHIFT}
+
+To run your {PRODUCT} SpringBoot business application on {OPENSHIFT}, create an immutable image and push this image to your {OPENSHIFT} environment.
+
+ifdef::PAM[]
+Optionally, you can also use {CENTRAL} Monitoring to monitor the execution of business processes in your application.
+endif::PAM[]
+
+.Prerequisites
+
+. You developed a {PRODUCT} SpringBoot business application. For instructions about creating the application, see xref:bus-app-create_{context}[].
+. If necessary, you configured Spring security for the application. For instructions about configuring Spring security, see xref:bus-app-security-con_{context}[].
+. You completed other Spring configuration for the business application, as necessary. For instructions about Spring configuration for your business application, see
+. You are logged on to your {OPENSHIFT} environment using the `oc` command and the required project is active.
+ifdef::PAM[]
+. If you want to use {CENTRAL} Monitoring, you installed {CENTRAL} Monitoring using the operator. For instructions about installing {CENTRAL} Monitoring and other {PRODUCT} components using the operator, see {URL_DEPLOYING_ON_OPENSHIFT}#assembly-openshift-operator[_{DEPLOYING_OPENSHIFT_OPERATOR}_].
++
+[IMPORTANT]
+====
+You must configure {CENTRAL} Monitoring to use the controller startup strategy. To enable the controller strategy on {CENTRAL} Monitoring, in the *Console* tab of the operator configuration, add a `KIE_SERVER_CONTROLLER_OPENSHIFT_ENABLED` environment variable and set it to `false`.
```

Comment:
This is a bit confusing to me. What has "controller startup strategy" to do with the business-central?
In immutable deployments you would typically use LocalContainersStartupStrategy and not ControllerBasedStartupStrategy. Can you please clarify this?